web/src/fetchStreamUrl.js

```diff
@@ -10,6 +12,30 @@ async function fetchStreamUrl(claimName, claimId) {
     });
 }
 
+/**
+ * Direct URL to the content's bits without redirects.
+ *
+ * Move back to 'utils/web' when `fetchStreamUrl` is no longer needed.
+ *
+ * @param claim
+ */
+function generateContentUrl(claim) {
+  const streamUrl = (claim) => {
+    // Hardcoded version of fetchStreamUrl().
+    return `${PLAYER_SERVER}/api/v3/streams/free/${claim.name}/${claim.claim_id}`;
+  };
+
+  const value = claim?.value;
+  if (value?.source?.media_type && value?.source?.sd_hash) {
+    const fileExt = `.${Mime.extension(value.source.media_type)}`;
+    const sdHash = value.source.sd_hash.slice(0, 6);
+    return `${streamUrl(claim)}/${sdHash}${fileExt}`;
+  }
+
+  return streamUrl(claim);
```

Comment:
We'll probably always have sdhash, but not extension (some old files won't, and maybe if metadata is not proper. So add sdhash, and the default to .mp4 if no extension?

Reply:
Livestreams won't have either, not sure how those work...

Reply:
- Did the `.mp4` fallback initially, but would then need to take audio into account, so ended up with the "no file, no ext" URL as the final fallback, which would at least still serve the content, just not satisfy Google.  I'm ok either way, let me know.
- For livestreams and non-audio/video content, probably fine as it is up to the caller to not use the function (unless we want to handle it here too). We are skipping the GoogleVideo metadata for livestreams for now (I think it's a whole new set of tags; haven't researched).